### DISCUSSION TITLE
Next Release?

### DISCUSSION THREAD

**Discussion by t0w0i7ne** [Textualize/textual#807] [General]
I am looking to make a simple interactive application with a TUI.  Textual and Rich appear to be the best option out there (with second place going to [pytermgui](https://github.com/bczsalba/pytermgui)) with the exception being that the latest release is missing most of the compelling/useful features that have been made available in the CSS branch.

The input widgets and fixes/feature additions that exist on the CSS branch are all I anticipate needing for my personal application's needs.  I'd very much prefer to use the widgets provided by the library/package than to include small third-party packages that are likely to die/break once the main library comes to feature parity.

For development of my own personal application, I can point setuptools to the git repo/branch, but I can absolutely see myself wanting to make TUIs in a professional setting with this library as well.  In that case my applications would need to be put on air-gaped systems and my employer would require that the version I use is an official release from the author(s) (even if alpha/beta).

What is the current plan for the next release?

**Comment by willmcgugan** [comment]
October 24th
